packages/nimble-components/src/table-column/date-text/index.ts

```diff
@@ -31,6 +120,143 @@ export class TableColumnDateText extends TableColumnTextBase {
             sortOperation: TableColumnSortOperation.basic
         };
     }
+
+    protected formatChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customLocaleMatcherChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customWeekdayChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customEraChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customYearChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customMonthChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customDayChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customHourChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customMinuteChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customSecondChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customTimeZoneNameChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customFormatMatcherChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customHour12Changed(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customTimeZoneChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customCalendarChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customDayPeriodChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customNumberingSystemChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customDateStyleChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customTimeStyleChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    protected customHourCycleChanged(): void {
+        this.updateColumnConfig();
+    }
+
+    private updateColumnConfig(): void {
+        const columnConfig: TableColumnDateTextColumnConfig = {
+            formatter: this.createFormatter(this.format)
```

Comment:
nit: I don't see a reason to pass `this.format` to `this.createFormatter` since it can read `this.format` from its own implementation.